_about-us/eng/shuli.md

```diff
@@ -10,21 +10,11 @@ functionId: eng
 jobTitle: Software Engineer
 curProducts:
   - FormSG
-pastProducts:
-  - CheckFirst
-  - Temperature
   - Signify
+pastProducts:
+  - GovLogin
 accomplishments:
-  - Independently made a number of FormSG infrastructure improvements to
-    increase reliability and performance. These include building an end-to-end
-    production-ready backup for Cloudflare DNS + CDN using AWS Route53 and
-    Cloudfront, designing and setting up infrastructure to host and serve static
-    assets for React app, and setting up bastion host infrastructure
-  - Led the integration of FormSG with SingPass/CorpPass/MyInfo, including
-    completing the migration of CorpPass to OIDC and decommissioning
-    SingPass/CorpPass SAML support with no downtime
-  - Built up engineering features for Signify from scratch, including
-    integration with the Sign with SingPass API
+  - I worked on FormSG and Signify
```

Comment:
I think we want to have specific accomplishments in this list. Things staff are specifically proud of, rather than generic statements of working on such and such products. 

For the original content, people ops took the data from folk's working docs, but you can change that to whatever you are most proud of, independently of the perf angle of the working docs. 

For you perhaps something like:
```
- Devised and implemented a waiting room layer to protect FormSG from large spikes. The waiting room successfully supported large form launches for the NDP registration forms.
- Singe-handedly figured out the Secure Digital Signature format in pdf and implemented it in Signify
```

Or whatever else you want to mention of course, you own your content (minus some editor prerogative form the people ops folks of course).